src/main/support.js

```diff
@@ -15,7 +15,7 @@ const currentVersion = require("../../package.json").version;
 const coreOptions = require("./core-options").options;
 
 // [prettierx] quick workaround:
-const currentVersion = "2.3.0";
+const currentVersion = "2.3.1";
```

Comment:
I think this would be much less likely to get overlooked if we moved it to package.json, for example:

```js
const currentVersion = require("../../package.json")["prettier-version"];
```

```json
{
  "name": "prettierx",
  "version": "0.19.0-dev",
  "prettier-version": "2.3.1",
  "description": "prettierx - less opinionated code formatter fork of prettier",
  "bin": "./bin/prettierx.js",
  "repository": "brodybits/prettierx",
```

Reply:
👍